rollup.config.js

```diff
@@ -15,13 +15,24 @@ import {
 const cdnUrl    = 'https://cdn.jsdelivr.net/';
 const sourcemap = true;
 const outputDir = 'dist';
-const external  = [ ...Object.keys(peerDependencies||{}) ];
+const external  = [ 
+  ...Object.keys(peerDependencies||{}),
+  new RegExp(cdnUrl)
+];
+
 const outputFormatExtensions = {
   'umd' : '.umd.js',
   'esm' : packageType=='module' ? '.js' : '.mjs',
   'cjs' : packageType!='module' ? '.js' : '.cjs',
 }
-const plugins = [ vue() ];
+const aliasPlugin = alias({ entries : [
+  { find: /^cdn!(.*)/,  replacement:`${cdnUrl}$1` },
+  { find: /^css!cdn!(.*)/,  replacement:`css!${cdnUrl}$1` },
```

Comment:
use \b (word boundary) instead of ^ to have a generic `cdn!` handler 

```suggestion
  { find: /\bcdn!(.*)/,  replacement:`${cdnUrl}$1` },
```